Adrin-Signoff-PushButton

Adrin11 opened the pull request.

arc2448 commented:
Test

JesseTNRoberts commented on Documentation/Signoffs/PushButton/PBDetailDesign.md:
```diff
@@ -0,0 +1,180 @@
+
+The push button subsystem will be the process of pushing the stop button to stop the time, which will denote the completion of the course. The system will consist of a dc motor, bevel gears, a shaft, a bearing, a L298N motor driver module, and a paint roller that will swing out on the arm perpendicular to the platform of the robot. The main bevel gear will be driven by the dc motor and the arm th ...
+
+## Specifications and Restraints
```
These are generally good. However, a few are not specific enough while others are too specific (must be 12in and not less than 12 in).

The schematics can't be seen because they're too small.

JesseTNRoberts reviewed (commented):
Images etc. should be in a folder. This is cluttered.

JesseTNRoberts commented on Documentation/Signoffs/PushButton/PBDetailDesign.md:
```diff
@@ -0,0 +1,254 @@
+The DC motor has a gear reduction ratio to increase torque. The reason for this is we need more torque than we do rotor/shaft speed so that it ensures the shaft for the button pusher is turned. To determine what our load torque is of the gears and shaft we can leave or input voltage as at least 6 volts and change the inertial from the shaft to a higher value that to see the max load inertia befor ...
+0.9833 rad/second which is eqivalent to 59 rad/minute for 2 seconds. The reason the shaft needs to be at 2 seconds is because at because if the shaft turns at 59 rad/minute for one second then the output shaft angle will only rotate 45 degrees which means our pushbutton will not be touching the wall. If we average 59 rad/min for 2 seconds then it will rotate out 90 degrees. The Matlab code that p ...
+
+**Matlab code:** 
```
Use a code block for snippets like this so that it displays well

JesseTNRoberts reviewed (commented):
There's a lot here. But I don't see an electrical schematic or a buildable 3d schematic.

Adrin11 commented:
Hey professor Roberts,

I don't know how my buildable schematic got removed but I will be re-adding it to the GitHub once I add my bracket I made for the arm to mount into.

Thanks,
Adrin Jackson
________________________________
From: JesseTNRoberts ***@***.***>
Sent: Saturday, December 2, 2023 11:12 AM
To: cebttu/CapstoneTeam1 ***@***.***>
Cc: Jackson, Adrin (asjackson42) ***@***.***>; Author ***@***.***>
Subject: Re: [cebttu/CapstoneTeam1] Adrin-Signoff-PushButton (PR #4)


External Email Warning

This email originated from outside the university. Please use caution when opening attachments, clicking links, or responding to requests.

________________________________

@JesseTNRoberts commented on this pull request.

There's a lot here. But I don't see an electrical schematic or a buildable 3d schematic.

—
Reply to this email directly, view it on GitHub<https://github.com/cebttu/CapstoneTeam1/pull/4#pullrequestreview-1760926674>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BDVM6EXAIJIR3Y5MEVE4QW3YHNOQNAVCNFSM6AAAAAA6Z5JPDCVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTONRQHEZDMNRXGQ>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

Adrin11 commented:
I see what happened, I moved it to the images folder in the GitHub directory and it forgot to update the new link to show the image in the report.

________________________________
From: Jackson, Adrin (asjackson42) ***@***.***>
Sent: Saturday, December 2, 2023 1:46 PM
To: cebttu/CapstoneTeam1 ***@***.***>; cebttu/CapstoneTeam1 ***@***.***>
Cc: Author ***@***.***>
Subject: Re: [cebttu/CapstoneTeam1] Adrin-Signoff-PushButton (PR #4)

Hey professor Roberts,

I don't know how my buildable schematic got removed but I will be re-adding it to the GitHub once I add my bracket I made for the arm to mount into.

Thanks,
Adrin Jackson
________________________________
From: JesseTNRoberts ***@***.***>
Sent: Saturday, December 2, 2023 11:12 AM
To: cebttu/CapstoneTeam1 ***@***.***>
Cc: Jackson, Adrin (asjackson42) ***@***.***>; Author ***@***.***>
Subject: Re: [cebttu/CapstoneTeam1] Adrin-Signoff-PushButton (PR #4)


External Email Warning

This email originated from outside the university. Please use caution when opening attachments, clicking links, or responding to requests.

________________________________

@JesseTNRoberts commented on this pull request.

There's a lot here. But I don't see an electrical schematic or a buildable 3d schematic.

—
Reply to this email directly, view it on GitHub<https://github.com/cebttu/CapstoneTeam1/pull/4#pullrequestreview-1760926674>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BDVM6EXAIJIR3Y5MEVE4QW3YHNOQNAVCNFSM6AAAAAA6Z5JPDCVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTONRQHEZDMNRXGQ>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

Adrin11 commented:
Hey professor Roberts,

I am just reaching out to let you know I made the changes to show how the arm will be mounted to the robot and have made a bracket that is 3-d printable with screw holes to mount onto the robot platform for the pushbutton subsystem. I also added the calculated lateral force on the arm of the robot for when it gets closer to the wall than expected. I also added the depiction for how the drive (mating)  gear will attach to the dc motor and the description of how the arm is connected to the shaft of the driven gear. If there is anything else that needs to be added or taken away, please let me know.

Thank you for your time,

Adrin Jackson
________________________________
From: Jackson, Adrin (asjackson42) ***@***.***>
Sent: Saturday, December 2, 2023 1:55 PM
To: cebttu/CapstoneTeam1 ***@***.***>; cebttu/CapstoneTeam1 ***@***.***>
Cc: Author ***@***.***>
Subject: Re: [cebttu/CapstoneTeam1] Adrin-Signoff-PushButton (PR #4)

I see what happened, I moved it to the images folder in the GitHub directory and it forgot to update the new link to show the image in the report.

________________________________
From: Jackson, Adrin (asjackson42) ***@***.***>
Sent: Saturday, December 2, 2023 1:46 PM
To: cebttu/CapstoneTeam1 ***@***.***>; cebttu/CapstoneTeam1 ***@***.***>
Cc: Author ***@***.***>
Subject: Re: [cebttu/CapstoneTeam1] Adrin-Signoff-PushButton (PR #4)

Hey professor Roberts,

I don't know how my buildable schematic got removed but I will be re-adding it to the GitHub once I add my bracket I made for the arm to mount into.

Thanks,
Adrin Jackson
________________________________
From: JesseTNRoberts ***@***.***>
Sent: Saturday, December 2, 2023 11:12 AM
To: cebttu/CapstoneTeam1 ***@***.***>
Cc: Jackson, Adrin (asjackson42) ***@***.***>; Author ***@***.***>
Subject: Re: [cebttu/CapstoneTeam1] Adrin-Signoff-PushButton (PR #4)


External Email Warning

This email originated from outside the university. Please use caution when opening attachments, clicking links, or responding to requests.

________________________________

@JesseTNRoberts commented on this pull request.

There's a lot here. But I don't see an electrical schematic or a buildable 3d schematic.

—
Reply to this email directly, view it on GitHub<https://github.com/cebttu/CapstoneTeam1/pull/4#pullrequestreview-1760926674>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BDVM6EXAIJIR3Y5MEVE4QW3YHNOQNAVCNFSM6AAAAAA6Z5JPDCVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTONRQHEZDMNRXGQ>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

JesseTNRoberts reviewed (commented):
The arm does not seem to be compressible? How does this keep pressure on the wall? 

The mechanical design will have to be sufficient so that we can actually build what you design. 

This has a lot of information but it is not put together. How do these pieces fit together in a single mechanical design? Why are there logic circuit diagrams for the internals of a device?

Adrin11 commented:
I mean I could just add a spring within the arm that will allow it to compress if we get too close to the wall, so I will add that to the subsystem. I will remove the logic circuit diagrams. Those diagrams were in there since the beginning of posting this subsystem. Did not know that would be an issue, but I will remove them and try to make the document more organized.

Thanks,
Adrin
________________________________
From: JesseTNRoberts ***@***.***>
Sent: Thursday, December 7, 2023 2:14 PM
To: cebttu/CapstoneTeam1 ***@***.***>
Cc: Jackson, Adrin (asjackson42) ***@***.***>; Author ***@***.***>
Subject: Re: [cebttu/CapstoneTeam1] Adrin-Signoff-PushButton (PR #4)


External Email Warning

This email originated from outside the university. Please use caution when opening attachments, clicking links, or responding to requests.

________________________________

@JesseTNRoberts commented on this pull request.

The arm does not seem to be compressible? How does this keep pressure on the wall?

The mechanical design will have to be sufficient so that we can actually build what you design.

This has a lot of information but it is not put together. How do these pieces fit together in a single mechanical design? Why are there logic circuit diagrams for the internals of a device?

—
Reply to this email directly, view it on GitHub<https://github.com/cebttu/CapstoneTeam1/pull/4#pullrequestreview-1770963054>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BDVM6ERYB553O5XVLT2NWUTYIIPTLAVCNFSM6AAAAAA6Z5JPDCVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTONZQHE3DGMBVGQ>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

JesseTNRoberts reviewed (commented):
Is this the only system that is using this battery and microcontroller? If not, the schematic should build on the existing schematics (or they build on this one) so that everything is shown. Otherwise, how do you know that pin is open?

JesseTNRoberts reviewed (commented):
How do we know that the motor will work off the output described (not all can generate a pwm).

JesseTNRoberts reviewed (commented):
current
